src/Resources/config/services.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <imports>
+        <import resource="services/event_listener.xml" />
+        <import resource="services/factory.xml" />
+        <import resource="services/fixture.xml" />
+        <import resource="services/form.xml" />
+        <import resource="services/menu.xml" />
+    </imports>
+
+    <services>
```

Comment:
I don't think you should have services in the same file that imports service files.

Reply:
Ok